### PR TITLE
Create zero-trust-architecture.md

### LINKED ISSUE #2326: [hi] Create zero-trust-architecture.md
There is no Zero-trust-architecture.md file for /hi localization

### PR DESCRIPTION
**Pull request by ARYANis0001**
### Describe your changes
Zero-trust-architecture.md file for /hi localization

### Related issue number or link 
(resolves #2326)
(https://github.com/cncf/glossary/issues/2326)

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by ARYANis0001** [comment]
Reviewers Please look on this PR

File: content/hi/content/en/zero-trust-architecture.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+शीर्षक: जीरो ट्रस्ट आर्किटेक्चर
+स्थिति: पूर्ण
+श्रेणी: संकल्पना
+टैग: ["सुरक्षा", "", ""]
```
Comment (aj11anuj) [inline content/hi/content/en/zero-trust-architecture.md:5]:
```suggestion
title: जीरो ट्रस्ट आर्किटेक्चर (Zero Trust Architecture)
status: पूर्ण
category: संकल्पना
tags: ["सुरक्षा", "", ""]
```

File: content/hi/content/en/zero-trust-architecture.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+---
+
+## यह क्या है
+
+जीरो ट्रस्ट आर्किटेक्चर आईटी सिस्टम के डिजाइन और कार्यान्वयन के लिए एक दृष्टिकोण निर्धारित करता है
```
Comment (aj11anuj) [inline content/hi/content/en/zero-trust-architecture.md:10]:
```suggestion
'जीरो ट्रस्ट आर्किटेक्चर' आई-टी सिस्टम के डिजाइन और कार्यान्वयन के लिए एक दृष्टिकोण निर्धारित करता है
```

File: content/hi/content/en/zero-trust-architecture.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+जहां भरोसा पूरी तरह से खत्म हो जाता है.
```
Comment (aj11anuj) [inline content/hi/content/en/zero-trust-architecture.md:11]:
```suggestion
जहां भरोसा पूरी तरह से खत्म हो जाता है।
```

File: content/hi/content/en/zero-trust-architecture.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+मूल सिद्धांत है "कभी भरोसा मत करो, हमेशा सत्यापित करो", डिवाइस या सिस्टम स्वयं,
```
Comment (aj11anuj) [inline content/hi/content/en/zero-trust-architecture.md:12]:
```suggestion
मूल सिद्धांत यह है की डिवाइस हो या सिस्टम "कभी भरोसा मत करो, हमेशा स्वयं पुष्टि करो"। 
```

File: content/hi/content/en/zero-trust-architecture.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+सिस्टम के अन्य घटकों से संचार करते समय, ऐसा करने से पहले हमेशा स्वयं को सत्यापित करें।
```
Comment (aj11anuj) [inline content/hi/content/en/zero-trust-architecture.md:13]:
```suggestion
सिस्टम के अन्य घटकों से संचार करते समय, ऐसा करने से पहले हमेशा स्वयं पुष्टि करें।
```

File: content/hi/content/en/zero-trust-architecture.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+आज कई नेटवर्कों में, कॉर्पोरेट नेटवर्क के भीतर, सिस्टम और डिवाइस एक-दूसरे के साथ स्वतंत्र रूप से संचार कर सकते हैं
+क्योंकि वे कॉर्पोरेट नेटवर्क परिधि की विश्वसनीय सीमा के भीतर हैं।
+जीरो ट्रस्ट आर्किटेक्चर विपरीत दृष्टिकोण अपनाता है, हालांकि नेटवर्क परिधि के अंदर,
+किसी भी संचार से पहले सिस्टम के घटकों को पहले सत्यापन से गुजरना पड़ता है।
```
Comment (aj11anuj) [inline content/hi/content/en/zero-trust-architecture.md:17]:
```suggestion
आज कई नेटवर्कों में, कंपनी के नेटवर्क के भीतर, सिस्टम और डिवाइस एक-दूसरे के साथ स्वतंत्र रूप से संचार कर सकते हैं, 
क्योंकि वे कंपनी के नेटवर्क परिधि की विश्वसनीय सीमा के भीतर हैं। 
जीरो ट्रस्ट आर्किटेक्चर विपरीत दृष्टिकोण अपनाता है, हालांकि नेटवर्क परिधि के अंदर,
किसी भी संचार से पहले सिस्टम के घटकों को पहले जाँच से गुजरना पड़ता है।
```

File: content/hi/content/en/zero-trust-architecture.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+## यह समस्या का समाधान करता है
+
+पारंपरिक विश्वास आधारित दृष्टिकोण के साथ जहां सिस्टम और उपकरण कॉर्पोरेट नेटवर्क परिधि के भीतर मौजूद होते हैं,
+धारणा यह है कि चूँकि विश्वास है, इसलिए कोई समस्या नहीं है।
```
Comment (aj11anuj) [inline content/hi/content/en/zero-trust-architecture.md:22]:
```suggestion
पारंपरिक विश्वास आधारित दृष्टिकोण के साथ जहां सिस्टम और उपकरण कंपनी के नेटवर्क परिधि के भीतर मौजूद होते हैं,
धारणा यह है कि "चूँकि विश्वास है, इसलिए कोई समस्या नहीं है"।
```

File: content/hi/content/en/zero-trust-architecture.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+हालाँकि, शून्य विश्वास वास्तुकला यह मानती है कि विश्वास एक भेद्यता है।
```
Comment (aj11anuj) [inline content/hi/content/en/zero-trust-architecture.md:23]:
```suggestion
हालाँकि, जीरो ट्रस्ट आर्किटेक्चर यह मानती है कि विश्वास एक भेद्यता है।
```

File: content/hi/content/en/zero-trust-architecture.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+ऐसी स्थिति में जहां किसी हमलावर ने किसी विश्वसनीय डिवाइस तक पहुंच प्राप्त कर ली हो,
+उस उपकरण को दिए गए विश्वास और पहुंच के स्तर पर निर्भर करता है,
+सिस्टम अब हमले के प्रति संवेदनशील है
```
Comment (aj11anuj) [inline content/hi/content/en/zero-trust-architecture.md:26]:
```suggestion
उस उपकरण को दिए गए विश्वास और पहुंच के स्तर पर निर्भर करता है कि,
सिस्टम अब हमले के प्रति कितना संवेदनशील है।
```

File: content/hi/content/en/zero-trust-architecture.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+चूंकि हमलावर "विश्वसनीय" नेटवर्क परिधि के भीतर है और पूरे सिस्टम में पार्श्व रूप से स्थानांतरित करने में सक्षम है।
+शून्य विश्वास वास्तुकला में, विश्वास को हटा दिया जाता है, जिससे हमले की सतह कम हो जाती है
+क्योंकि एक हमलावर को अब पूरे सिस्टम में आगे जाने से पहले सत्यापित करने के लिए मजबूर किया जाता है।
```
Comment (aj11anuj) [inline content/hi/content/en/zero-trust-architecture.md:29]:
```suggestion
जीरो ट्रस्ट आर्किटेक्चर में, विश्वास को हटा दिया जाता है, जिससे हमले की सतह कम हो जाती है
क्योंकि एक हमलावर को अब पूरे सिस्टम में आगे जाने से पहले जाँच करने के लिए मजबूर किया जाता है।
```

File: content/hi/content/en/zero-trust-architecture.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+
+## यह कैसे मदद करता है
+
+शून्य विश्वास वास्तुकला को अपनाने से बढ़ी हुई सुरक्षा का मुख्य लाभ मिलता है
+हमले की सतह में कमी के साथ.
+आपके कॉर्पोरेट सिस्टम से भरोसा हटाने से अब सुरक्षा द्वारों की संख्या और ताकत बढ़ जाती है
+एक हमलावर को सिस्टम के अन्य क्षेत्रों तक पहुंच प्राप्त करने के लिए इससे गुजरना पड़ता है।
```
Comment (aj11anuj) [inline content/hi/content/en/zero-trust-architecture.md:36]:
```suggestion
जीरो ट्रस्ट आर्किटेक्चर को अपनाने से बढ़ी हुई सुरक्षा का मुख्य लाभ यह मिलता है कि, 
हमले की सतह में कमी के साथ 
आपके कंपनी के सिस्टम से भरोसा हटाने से अब सुरक्षा द्वारों की संख्या और ताकत दोनों बढ़ जाती है। 
एक हमलावर को सिस्टम के अन्य क्षेत्रों तक पहुंच प्राप्त करने के लिए इससे गुजरना पड़ता है।
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]
Some suggestions from my side

**Review by kumarankit999 — COMMENTED** [review]
Please Improve this PR, as suggested by @aj11anuj

**Comment by ARYANis0001** [comment]
Sure

**Comment by ARYANis0001** [comment]
I will improve

**Review by ARYANis0001 — COMMENTED** [review]
Updated

**Comment by ARYANis0001** [comment]
Please review this I have made the changes

File: content/hi/content/en/zero-trust-architecture.md [hunk 12]
```diff
@@ -0,0 +1,36 @@
+---
+title: जीरो ट्रस्ट आर्किटेक्चर (Zero Trust Architecture)
+status: पूर्ण
```
Comment (aj11anuj) [inline content/hi/content/en/zero-trust-architecture.md:3]:
```suggestion
status: completed
```

File: content/hi/content/en/zero-trust-architecture.md [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+category: संकल्पना
```
Comment (aj11anuj) [inline content/hi/content/en/zero-trust-architecture.md:4]:
```suggestion
category: concept
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]
Apart from these minor suggestions, rest of the things looks good to me 👍

**Review by ARYANis0001 — COMMENTED** [review]
Updated

**Comment by ARYANis0001** [comment]
Please review this I have made the changes

**Review by kumarankit999 — COMMENTED** [review]
lgtm if you have gone through @aj11anuj Review!

**Review by aj11anuj — APPROVED** [review]
Lgtm 👍

**Review by kumarankit999 — COMMENTED** [review]
looks good now!

**Review by kumarankit999 — APPROVED** [review]
lgtm

**Comment by ARYANis0001** [comment]
@kumarankit999 how i can contact to you

**Comment by ARYANis0001** [comment]
ok wait

**Comment by ARYANis0001** [comment]
@aj11anuj  I have changed the branch to cncf dev-hi

**Review by aj11anuj — COMMENTED** [review]
I just noticed it, Location of this file "content/hi/content/en/zero-trust-architecture.md" doesn't exist. It should be "content/hi/zero-trust-architecture.md". And due to recent sync of dev-hi branch with main branch since a long time, some issues are arising like this one- "Zero-trust-architecture file already exists". So, now we cannot accept this one. Sorry for this inconvenience.

**Comment by ARYANis0001** [comment]
Ok ki will close this

**Pull request closed without merging**